tests/unit/drivers/test_vector_index_driver.py

```diff
@@ -11,6 +11,9 @@
 from jina.types.document import Document
 
 
+# TODO: Add tests for sparse vectors index driver.
```

Comment:
just a reminder

Reply:
@bwanglzu I have done changes to avoid having extra `Driver` classes, so that it is easier for the user to use.

Still the `VectorIndexDriver` and `VectorSearchDriver` need to be unit tested